src/EditorFeatures/Core.Wpf/NavigableSymbols/NavigableSymbolService.NavigableSymbol.cs

```diff
@@ -58,25 +46,21 @@ private class NavigableSymbol : INavigableSymbol
             public void Navigate(INavigableRelationship relationship)
             {
                 // Fire and forget.
-                var token = _listener.BeginAsyncOperation(nameof(NavigateAsync));
+                var token = _service._listener.BeginAsyncOperation(nameof(NavigateAsync));
                 _ = NavigateAsync().ReportNonFatalErrorAsync().CompletesAsyncOperation(token);
             }
 
             private async Task NavigateAsync()
             {
-                using var context = _uiThreadOperationExecutor.BeginExecute(
+                using var context = _service._uiThreadOperationExecutor.BeginExecute(
```

Comment:
talked offline - could use the background wait indicator here since we already use it for gotodef

Reply:
done.